types/src/front/assistant/templates.ts

```diff
@@ -113,6 +121,37 @@ export const ActionPresetCodec = ioTsEnum<ActionPreset>(
   Object.keys(ACTION_PRESETS),
   "ActionPreset"
 );
+
+// MULTI ACTION MODE
+
+type MultiActionType =
```

Comment:
Not great to have this duplicated here but we have it here for `presetAction` too. 
Deserves a refactor -> we would need to move the types used in the app from front to types. 
I don't think it's a blocker to ship this since this was already duplicated.